model-integration/src/main/java/ai/vespa/generative/LanguageModelGenerator.java

```diff
@@ -0,0 +1,29 @@
+package ai.vespa.generative;
```

Comment:
Same, I think we could put this in llm.generation

And maybe, since what separates it from Generator isn't that it's generating from a language model, but that it's taking an additional config specifying a prompt, we could call it something like PromptedGenerator, or ConfiguredGenerator?